.github/workflows/lintern.yml

```diff
@@ -0,0 +1,13 @@
+name: Lintern
+on: [push]
+jobs:
+  lintern:

```

Comment:
@alberturria what do we mean by `lintern`?

Reply:
Woopsie! I thought that was a word. I was referring to the fact of `linting` the code. (Find stylistic errors, in our case, according to the rules defined in the `eslintrc.json` file).

Are you OK with changing the name of the file, action, and job to `Lint`, or do you have any other suggestion? 🙂

Reply:
No problem, and I think either `build`, `lint` or both would be fine 👍